lib/rpmtriggers.c

```diff
@@ -99,23 +99,32 @@ static void rpmtriggersSortAndUniq(rpmtriggers trigs)
     }
 }
 
+static unsigned int getTrigPriority(Header h, rpmTagVal prioTag, int tix)
+{
+    unsigned int priority = 1000000;
```

Comment:
This looks like a magic number that should be put into a preprocessor macro...?

Reply:
It should, yeah. Will do.

Reply:
... because it's also used in `runImmedFileTriggers()` now (otherwise it would be fine as a local variable of course).

Reply:
Turns out, this priority number is also used in `parseScript()` :smile:

Reply:
Oh, I didn't refresh the page before adding more comments... Good then :smile:

Reply:
...and done.